File: CouponSystem/src/main/java/com/CouponSystem/CouponSystem/Entities/LoginRequest.java
package com.CouponSystem.CouponSystem.Entities;

public class LoginRequest {
	
	public String name;
	public String password;
	public String type;
	@Override
	public String toString() {
		return "LoginRequest [name=" + name + ", password=" + password + ", type=" + type + "]";
	}
	
	
}

File: AngularReStudy/AngularProject/src/app/Models/login-request.ts
export class LoginRequest {
  private name: string;
  private password: string;
  private type: string;

  setType(type: string) {
    this.type = type;
  }

  getType() {
    return this.type;
  }

  setName(userName) {
    this.name = userName;
  }

  getName() {
    return this.name;
  }

  setPassword(password) {
    this.password = password;
  }

  getPassword() {
    return this.password;
  }
}
